### ISSUE TITLE
Add Table

### ISSUE THREAD

**Issue by jolbol1** [jolbol1/jolly-ui#64]
At this time, Table is the only component missing from React-Aria-Components.

Should mirror https://ui.shadcn.com/docs/components/table but with the RAC components.
